### DISCUSSION TITLE
character's saving quirks / inventory loss

### DISCUSSION THREAD

**Discussion by DrBruhman** [FakeFishGames/Barotrauma#14397] [Bug reports]
i was husk with zealote robe, i hided outside of outpost and left the server. after next 2 days and i was greeted by default start items and empty mouth on same save, while talents stayed the same. this also happened to our doctor(was not husk and rejoined the same day). 

reproduce: set up multiplayer server, make client character,  be on outpost, disconnect client, probably wait for brain death, save game by sailing off or saving in esc menu, after loading save connect client and observe lost items, afflictions.

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14298
